packages/mux-player/test/player.test.js

```diff
@@ -594,24 +597,22 @@ describe('<mux-player>', () => {
     });
 
     (isSafari ? it.skip : it)('should have something in the buffer if canplay', async function () {
-      this.timeout(5000);
+      this.timeout(10000);
```

Comment:
NOTE: Some of these tests still fail when running remotely on Sauce Labs. Unclear what the root cause is, but I've written functionally equivalent code for live browser testing (via remote dev tools) on Win11 + Chrome v130 and it works. Treating as out of scope.